subsys/net/ip/net_shell.c

```diff
@@ -1645,7 +1645,7 @@ static int cmd_net_dns_query(const struct shell *shell, size_t argc,
 {
 
 #if defined(CONFIG_DNS_RESOLVER)
-#define DNS_TIMEOUT K_MSEC(2000) /* ms */
+#define DNS_TIMEOUT (MSEC_PER_SEC * 2) /* ms */
```

Comment:
Check the trailing comment please.

Reply:
What is there to check? The DNS_TIMEOUT is in milliseconds.

Reply:
> What is there to check? The DNS_TIMEOUT is in milliseconds.

Hmm, ok. I would guess that `/* ms */` comment is a left-over from old time when there was a raw number, like `2000 /* ms */`. Thus, I hinted at removing it, as in other places where there're unit-providing macros, we don't use it. But yes, formally, it's still in ms.